### ISSUE TITLE
What is the suggested training step?

### ISSUE THREAD

**Issue by exeex** [syang1993/FFTNet#4]
Hello,

This is a awesome implement of FFTNet with clear instruction to install.

However I found the loss would stuck around 3.0xxx on cmu_us_slt_arctic dataset：

step [7647]: loss: 3.211
step [7648]: loss: 3.155
step [7649]: loss: 3.065
step [7650]: loss: 3.126

I would like to know what is the suggested training step on cmu_us_slt_arctic dataset.

thanks,

**Comment by yoyolicoris** [comment]
Maybe you can try run more steps and increase the learning rate. The final loss should be around 2.6 in my experience.
